### ISSUE TITLE
Custom HTML block: line numbers lack sufficient contrast

### ISSUE THREAD

**Issue by sarahmonster** [WordPress/gutenberg#8736]
Line numbers may not be considered critical to the user experience, but with the current shade of grey, they don't have sufficient contrast with the white background:

<img width="669" alt="screenshot 2018-08-08 23 56 28" src="https://user-images.githubusercontent.com/376315/43868694-b2ebc1d6-9b66-11e8-8608-36ddf0a63456.png">

> The color combination #999999/#ffffff has a contrast ratio of 2.85, which is not sufficient. At this size, you will need a ratio of at least 4.5. 

<img width="641" alt="screenshot 2018-08-08 23 55 20" src="https://user-images.githubusercontent.com/376315/43868692-b2a7e736-9b66-11e8-84a0-930405d557eb.png">

**Comment by earnjam** [comment]
Created a core ticket for this: https://core.trac.wordpress.org/ticket/44763

**Comment by abdullah1908** [comment]
@earnjam patch updated on core track regarding this fix :+1: 
https://core.trac.wordpress.org/ticket/44763#comment:4

**Comment by nerrad** [comment]
It doesn't look like the html block uses code mirror anymore (and there are no line numbers so I'm going to close this.
